citi-onboarding/maracatronics
# Feature/who we are model

## What i did
- Created the Who we are model;
- Created the Who we are controller;
- Created the Who we are route;

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/who-we-are-model" with ```git checkout feature/who-we-are-model```
3. Now, you need to found the server folder with ```cd server```
4. On the server folder, run the server with ```node index.js``` and go to http://localhost:3001/admin
5. Go to "Quem somos" model, edit the text and put some image
6. Now, go to the mongoDB and Cloudinary account and check if the image are there and if the text changed.
7. On mongoDB account, click in Maraca and, after, in Collections to see the data. 

_The accounts will be in Notion._
